Code/Framework/AzToolsFramework/AzToolsFramework/UI/PropertyEditor/PropertyBoolComboBoxCtrl.hxx

```diff
@@ -42,6 +43,11 @@ namespace AzToolsFramework
         QWidget* GetLastInTabOrder();
         void UpdateTabOrder();
 
+        QComboBox* GetComboBox();
+        QToolButton* GetEditButton();
+
+        void SetEditButtonCallBack(AZ::Edit::AttributeFunction<bool(bool)>* function);
```

Comment:
nit: It'd be nice to use an alias for `AZ::Edit::AttributeFunction<bool(bool)>`, it'd make it more readable in the places it's used.